phase0/walkthrough.md

```diff
@@ -0,0 +1,3 @@
+A user wishes to create a game showing the branching paths of a study session. To do so, they first run the GUIBoxScene class, which contains the main method for the engine. Once GUIBoxScene opens they can choose to Add Slide (which creates a new instance of a GUIScene object) or Add Decision (which creates a new instance of a GUIDecision object attached to and resulting in a GUIScene instance). Together they combine these options to create a flowchart for their game. Once they have a finished product they click the “Save” button, which creates an instance of the RDFSave class that saves their game to a .trig file.
+
+With their game saved they can now open it through the Player class main method. The game will begin with the slide they designated as first in the engine, and then proceed based on what decisions they choose. They will be able to see the decisions and slides they created reflected in the game they are playing, except now as the Slide and Decision classes instead of GUIDecision and GUIScene. If they were to reopen the engine they could use the RDFLoad button to load the game they were working on before and keep editing it.
```

Comment:
The game now opens throught the game renderer class main method as I didn't want to have an import of the game renderer class in player as that breaks clean architecture